ftplugin/yagpdbcc/functions.vim

```diff
@@ -11,7 +11,6 @@
 " but WITHOUT ANY WARRANTY; without even the implied warranty of
 " MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 " GNU General Public License for more details.
-
```

Comment:
Just for formatting consistency, license headers in other files do retain this blank line...